### ISSUE TITLE
Semantic Entropy calculation

### ISSUE THREAD

**Issue by manhitv** [ml-jku/SDLG#2]
Hi team,

Thanks for publishing your implementation. 

I have a question when calculating Semantic Entropy from your code (utils.compute_semantic_entropy, line 744). I guess it would be as follows based on probability, not likelihood.

`entropy = - torch.sum(torch.exp(aggregated_normalized_likelihoods) * aggregated_normalized_likelihoods, dim=0)`
